src/app/services/authors-store.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, finalize, Observable, tap } from 'rxjs';
+import { Author } from '../app-model';
+import { AuthorsService } from './authors.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthorsStoreService {
+  private authors$$ = new BehaviorSubject<Author[]>([]);
+  authors$: Observable<Author[]> = this.authors$$.asObservable();
+
+  isLoading$ = new BehaviorSubject<boolean>(false);
```

Comment:
You also should create `isLoading Observable` by the task, isn't it?

Reply:
added